common/src/main/webapp/sass/tooltips.scss

```diff
@@ -42,7 +42,7 @@
 }
 
 .svg-target-tooltip-area {
-  pointer-events: bounding-box;
```

Comment:
I'm wondering if there's some sort of s?css linting we can set up that warns for non-standard usage

Reply:
Yeah that would be useful